resources/JenkinsfileTemplate.groovy

```diff
@@ -262,7 +262,7 @@ pipeline {
               warnError('installTools failed')
             }
             steps {
-              installTools([ [tool: 'yq', version: '3.3' ] ])
+              installTools([ [tool: 'yq', version: '3.3.4' ] ])
```

Comment:
It's not the version, as the `3.3` is resolved as `3.3.4`

Reply:
`installTools` uses the versions >= `3.3` so the patch versions are searched and gather the newest one that fulfils the version match.

Reply:
IIRC, `yq` is not required but only the validation of the `installTools` step:
- https://github.com/elastic/apm-pipeline-library/pull/804

`nodejs` was the initial implementation, but with the new versions then it did not work well, so I probably swapped to `yq` to use something more stable.

Maybe for simplicity, we can forget about the `installTools` step and remove it, or use another tool

Reply:
I agree, we could use another tool that we already know is present in all package managers for platform